### PR TITLE
User authentication on client side for creating/edit article

### PR DESCRIPTION
**Pull request by Fraseer**
PT Link: https://www.pivotaltracker.com/story/show/180127381

## User Story
As a news site
In order to allow journalist to upload articles
I would like to provide them with the authorization to do so

## Changes Proposed
- Add j-tock-auth
- Create login view on render
- Include authorization in axios calls

## Screenshots
image

### CODE REVIEW

File: src/modules/apiHelper.js [hunk 1]
```diff
@@ -18,7 +19,8 @@ const Article = {
   },
 
   async show(articleId) {
-    let response = await axios.get(`/api/articles/${articleId}`);
+    const headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"))
```
Comment (dorianbuck) [inline src/modules/apiHelper.js:22]: How does this work?

File: src/modules/auth.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+let url = "";
+if (!process.env.NODE_ENV || process.env.NODE_ENV === "development") {
+  url = "http://localhost:3000";
+} else {
+  url = "https://news-in-progress-api.herokuapp.com";
+}
+const auth = new JtockAuth({
+  host: url,
+  prefixUrl: "/api",
+});
```
Comment (dorianbuck) [inline src/modules/auth.js:12]: 👍🏽

**Review by dorianbuck — COMMENTED** [review]
Great work like always. Curious question, is the logo pink or orange?? Either way I like it 😄

**Comment by Fraseer** [comment]
> Great work like always. Curious question, is the logo pink or orange?? Either way I like it 😄 

It seems to change when I take a screenshot of it. It's def more red than pink when you run the app, but then in a screenshot it's really florescent pink for some reason🤷🏻‍♂️

**Pull request merged**